### ISSUE TITLE
Split documentation into Moya usage and others

### ISSUE THREAD

**Issue by BasThomas** [Moya/Moya#1143]
See https://github.com/Moya/Moya/pull/1142#issuecomment-310930345

**Comment by BasThomas** [comment]
Any idea as to what to call the "others"?

**Comment by sunshinejr** [comment]
I thought about it for a while, but couldn't get anything good enough. I came to a conclusion that we can just move `CommunityProjects.md`, `Development.md` & `Releasing.md` from `docs/` into the root directory for now (like with everything else that does not touch Moya usage itself). This idea is also supported by GitHub (they expect `Code of Conduct` & `Contributing` to be in the root dir as well).

**Comment by BasThomas** [comment]
This is a bit trickier now as the documents have been (partially) translated to Chinese and I am not a big fan of pulling both the English and the Chinese versions into the root.

**Comment by BasThomas** [comment]
... allthough we did the same already with the Readme, CoC and Vision. I feel like even further clouding the root is not a great idea.

**Comment by sunshinejr** [comment]
Yeah, we probably need to group it somehow. No idea how, though 😞

**Comment by Austinate** [comment]
Hello @sunshinejr and @BasThomas ! Just wanted to jump into discussion, if you don't mind 🙃

According to GitHub [readme](https://help.github.com/articles/about-readmes/) 
tl;dr: "If you put your README file in your repository's root, docs, or hidden .github directory, GitHub will recognize and automatically surface your README to repository visitors." 
So it looks like we can move current README's in 'docs' folder one level deeper and our root README.md can go to that folder. (or probably we should swift to '.github' folder).

btw, i see same info regarding [contribution guidelines](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) and [CoC](https://help.github.com/articles/adding-a-code-of-conduct-to-your-project/)
